tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1262,6 +1252,19 @@ def _save_calibration_table(node):
       self._converted_func = self._rebuild_func(self._converted_func)
 
     self._converted = True
+
+    graphviz_path = os.environ.get("TF_TRT_EXPORT_GRAPH_VIZ_PATH", default=None)
+    if graphviz_path is not None:
+      try:
+        trt_utils.export_graph_to_dot(
+          graphdef=self._converted_func.graph.as_graph_def(add_shapes=True),
+          dot_output_filename=graphviz_path
+        )
+      except Exception as e:
+        logging.error(
+            "An Exception occured during the export of the graph "
+            f"visualization: {str(e)}")
```

Comment:
I think "e" is enough to print the string of e, no need to use "str(e)".

Reply:
Fixed